modules/deploy/pages/deployment-option/cloud/api/cloud-dataplane-api.adoc

```diff
@@ -0,0 +1,138 @@
+= Manage Cluster Resources Using the Cloud API
```

Comment:
Also need confirmation on whether this page title is okay, or if it should be reworded (is "cluster resources" accurate?)